src/NUnitFramework/framework/Internal/TestProgressReporter.cs

```diff
@@ -64,14 +64,9 @@ public void TestStarted(ITest test)
             var parent = GetParent(test);
             try
             {
-                string report = string.Format(
-                    "<{0} id=\"{1}\" parentId=\"{2}\" name=\"{3}\" fullname=\"{4}\" type=\"{5}\"/>",
-                    startElement,
-                    test.Id,
-                    parent != null ? parent.Id : string.Empty,
-                    FormatAttributeValue(test.Name),
-                    FormatAttributeValue(test.FullName),
-                    test.TestType);
+                string report = test is TestSuite 
+                    ? $"<{startElement} id=\"{test.Id}\" parentId=\"{(parent != null ? parent.Id : string.Empty)}\" name=\"{FormatAttributeValue(test.Name)}\" fullname=\"{FormatAttributeValue(test.FullName)}\" type=\"{test.TestType}\"/>" 
+                    : $"<{startElement} id=\"{test.Id}\" parentId=\"{(parent != null ? parent.Id : string.Empty)}\" name=\"{FormatAttributeValue(test.Name)}\" fullname=\"{FormatAttributeValue(test.FullName)}\" type=\"{test.TestType}\" classname=\"{FormatAttributeValue(test.ClassName ?? "")}\" methodname=\"{FormatAttributeValue(test.MethodName ?? "")}\"/>";
```

Comment:
Looking at the entire `TestStarted` method I think we should remove `startElement` and just state the elements explicitly
```suggestion
                string report = test is TestSuite 
                    ? $"<start-suite id=\"{test.Id}\" parentId=\"{(parent != null ? parent.Id : string.Empty)}\" name=\"{FormatAttributeValue(test.Name)}\" fullname=\"{FormatAttributeValue(test.FullName)}\" type=\"{test.TestType}\"/>"
                    : $"<start-test id=\"{test.Id}\" parentId=\"{(parent != null ? parent.Id : string.Empty)}\" name=\"{FormatAttributeValue(test.Name)}\" fullname=\"{FormatAttributeValue(test.FullName)}\" type=\"{test.TestType}\" classname=\"{FormatAttributeValue(test.ClassName ?? "")}\" methodname=\"{FormatAttributeValue(test.MethodName ?? "")}\"/>";
```
also I'm considering whether we should wrap these lines, as the longest is more than 350 characters.